tpl/tplimpl/template_embedded.go

```diff
@@ -123,32 +123,47 @@ func (t *templateHandler) embedTemplates() {
 `)
 
 	t.addInternalTemplate("", "pagination.html", `{{ $pag := $.Paginator }}
-    {{ if gt $pag.TotalPages 1 }}
-    <ul class="pagination">
-        {{ with $pag.First }}
-        <li>
-            <a href="{{ .URL }}" aria-label="First"><span aria-hidden="true">&laquo;&laquo;</span></a>
-        </li>
-        {{ end }}
-        <li
-        {{ if not $pag.HasPrev }}class="disabled"{{ end }}>
-        <a href="{{ if $pag.HasPrev }}{{ $pag.Prev.URL }}{{ end }}" aria-label="Previous"><span aria-hidden="true">&laquo;</span></a>
-        </li>
-        {{ range $pag.Pagers }}
-        <li
-        {{ if eq . $pag }}class="active"{{ end }}><a href="{{ .URL }}">{{ .PageNumber }}</a></li>
-        {{ end }}
-        <li
-        {{ if not $pag.HasNext }}class="disabled"{{ end }}>
-        <a href="{{ if $pag.HasNext }}{{ $pag.Next.URL }}{{ end }}" aria-label="Next"><span aria-hidden="true">&raquo;</span></a>
-        </li>
-        {{ with $pag.Last }}
-        <li>
-            <a href="{{ .URL }}" aria-label="Last"><span aria-hidden="true">&raquo;&raquo;</span></a>
-        </li>
-        {{ end }}
-    </ul>
-    {{ end }}`)
+{{ if gt $pag.TotalPages 1 }}
+<ul class="pagination">
+    {{ with $pag.First }}
+    <li>
+        <a href="{{ .URL }}" aria-label="First"><span aria-hidden="true">&laquo;&laquo;</span></a>
+    </li>
+    {{ end }}
+    <li
+    {{ if not $pag.HasPrev }}class="disabled"{{ end }}>
+    <a href="{{ if $pag.HasPrev }}{{ $pag.Prev.URL }}{{ end }}" aria-label="Previous"><span aria-hidden="true">&laquo;</span></a>
```

Comment:
> ...would nice if you could post a link to those standards?

Sure:

1. [AT screen readers and `aria-current` (Video)](https://www.youtube.com/watch?v=o1FyUaaK8hk) (Note this also shows tab indexing and `aria-label`). 
2. [w3's `aria-current`, spec](https://www.w3.org/TR/wai-aria-1.1/h-aria-current); note section on pagination and `page` token. 
3.  [Accessible pagination (A11Y)](http://www.a11ymatters.com/pattern/pagination/)

> but I suspect those standards are written by people who see

* ["Myth: Accessibility is 'blind people'"](http://a11yproject.com/posts/myth-accessibility-is-blind-people/)

I agree we can table these as maybe "future iterative improvements?"